ci/closures.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+set -xe
+
+# Enables printing of all potential GC allocation sources to stdout
+export DFLAGS=-vgc
+
+# Prepare sources
+beaver dlang make d2conv
+
+# Ensure that there are no lines about closure allocations in the output
+! beaver dlang make fasttest 2>&1 | grep "closure"
```

Comment:
Only fast because this is detected at compile-time, right? So we only need to parse the actual library modules and not any unit-tests modules?

Reply:
Correct - and we get less false positives this way (we don't really care about closures in tests)